File: iOS/Pool/Sources/Pool/PoolGamblerScoreTransformer.swift
import DataPool

public extension PoolGamblerScore {
    func toPoolGamblerScoreModel() -> PoolGamblerScoreModel {
        PoolGamblerScoreModel(
            poolId: self.poolId,
            poolName: self.poolName,
            gamblerId: self.gamblerId,
            gamblerUsername: self.gamblerUsername,
            position: self.position,
            beforePosition: self.beforePosition,
            score: self.score
        )
    }
}

File: Android/pool/src/main/java/com/felipearpa/tyche/pool/PoolGamblerScoreTransformer.kt
package com.felipearpa.tyche.pool

import com.felipearpa.tyche.data.pool.domain.PoolGamblerScore

fun PoolGamblerScore.toPoolGamblerScoreModel() =
    PoolGamblerScoreModel(
        poolId = poolId,
        poolName = poolName,
        gamblerId = gamblerId,
        gamblerUsername = gamblerUsername,
        position = position,
        beforePosition = beforePosition,
        score = score,
        gamblerCount = gamblerCount,
    )
